sea-orm-macros/src/derives/active_enum.rs

```diff
@@ -37,43 +37,45 @@ impl ActiveEnum {
             ident_span => compile_error!("Missing macro attribute `db_type`");
         }));
         for attr in input.attrs.iter() {
-            if let Some(ident) = attr.path.get_ident() {
-                if ident != "sea_orm" {
-                    continue;
-                }
-            } else {
+            if !attr.path().is_ident("sea_orm") {
                 continue;
             }
             if let Ok(list) = attr.parse_args_with(Punctuated::<Meta, Comma>::parse_terminated) {
                 for meta in list.iter() {
                     if let Meta::NameValue(nv) = meta {
                         if let Some(name) = nv.path.get_ident() {
                             if name == "rs_type" {
-                                if let Lit::Str(litstr) = &nv.lit {
-                                    rs_type = syn::parse_str::<TokenStream>(&litstr.value())
-                                        .map_err(Error::Syn);
+                                if let Expr::Lit(exprlit) = &nv.value {
+                                    if let Lit::Str(litstr) = &exprlit.lit {
```

Comment:
The `value` of `MetaNameValue` is now `Expr`, so we need to check whether the value is `Lit` or not first.